Cinema Room Manager.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do you need this file on GitHub? if not - remove it

Reply:
No, there is no need in it on GitHub. File already removed